internal/ledgerutil/compare.go

```diff
@@ -406,46 +493,52 @@ func readMetadata(fpath string) (*kvledger.SnapshotSignableMetadata, error) {
 }
 
 // Compares hashes of snapshots
-func snapshotsComparable(fpath1 string, fpath2 string) (bool, string, uint64, error) {
+func hashesEqual(fpath1 string, fpath2 string) (bool, bool, string, uint64, error) {
```

Comment:
Much better name.
Now that the return is getting more complicated, I would expand the function comment to describe the return parameters. Alternatively, you can declare the return variables names here to make it more clear what will be returned.